tests/phpunit/tests/test-preload-paths.php

```diff
@@ -41,26 +42,26 @@ public function test_preload_paths_with_post_editor_context( $path, $is_filtered
 	/**
 	 * @dataProvider preload_paths_provider
 	 *
-	 * @param string|array $path            The preload path under test. Could be an array if provided along a HTTP method.
-	 * @param bool         $is_filtered     Whether the path should be filtered or not.
-	 * @param WP_Post      $post            The post provided for the context.
-	 * @param string       $language        The post's language slug.
-	 * @param bool         $is_translatable Whether or not the post type is translatable.
+	 * @param string|string[] $path         The preload path under test. Could be an array if provided along a HTTP method.
+	 * @param bool            $is_filtered  Whether the path should be filtered or not.
+	 * @param string          $context_post Says what type of post must be created for the context. Possible values are `translatable`, `translatable_cpt`, `untranslatable`.
+	 * @param string          $language     The post's language slug.
 	 */
-	public function test_preload_paths_in_site_editor_context( $path, $is_filtered, $post, $language, $is_translatable ) {
+	public function test_preload_paths_in_site_editor_context( $path, bool $is_filtered, string $context_post, string $language ) {
+		list( $path ) = $this->make_data_concrete( $path, $is_filtered, $context_post );
```

Comment:
I think `list()` must be avoided even in test.
Am I right @Chouby?
See also line 13.

Reply:
I think you're confusing `list()` with `extract()`: #1333.

Reply:
Sorry to insist, `list()` has been removed from both our code and tests I think. I didn't find any occurence of this function.

Reply:
253 matches in WP's core currently.

Reply:
The core isn't always well written IMHO 😅